Category ID Fix + Country Key

gjusticea opened the pull request.

gjusticea committed:
Category ID Fix

Fixed the category ID system for all scripts, entries, and util functions

gjusticea committed:
Create country_key.csv

nikosbosse commented on scripts/List of Category 5 Atlantic hurricanes.R:
```diff
@@ -2,13 +2,14 @@ source("utils/functions.R")
 # Table 2 is the one we want
 # Table 5 may be interesting if we want a better picture of damage
-table = tables[[2]] %>%
+table = tables[[1]] %>%
```
Did the relevant table change?

nikosbosse replied:
Haven't checked, just asking in case this was by accident

nikosbosse commented on scripts/[manual] List of recessions.R:
```diff
@@ -57,6 +56,15 @@ select(Category, Event, `Event description`, `Timepoint start`,
        `Timepoint end`, `Quantity outcome 1`, `Reference/link to data`,
        `Accessed on`,cc)
 
+cat_id_key = table_clean %>%
```
As a general note, maybe adding a bit more comments in the code for future you / me / others would be helpful

nikosbosse reviewed (approved):
Looks good to me!

gjusticea committed:
Cleaning for Sheets port

Various cleaning efforts to standardize file and data formatting, especially renaming every output csv to include "list of" in the title, and standardizing date formats as much as possible. Added script to combine all lists into one file to import to Google Sheets

gjusticea committed:
Country Key start

Created a new table mapping Category+Event name combinations to countries for applicable lists. Complete list is WIP

gjusticea committed:
US death sentences, exonerations, and executions

Added scripts, data, and metadata for two lists of US executions, and US death sentences labeled by whether the person was exonerated

gjusticea committed:
File type correction

Changing output from script to properly by .csv

gjusticea committed:
Coup attempts data

added script/data/metadata for coup attempts

gjusticea committed:
Potential future data

added data on coups, and potential future data sets from CSP

gjusticea committed:
Terrorist attacks

Added script/data/metadata on deadly terrorist attacks

gjusticea committed:
Solar flares

Added scripts/data/metadata on major solar flares

gjusticea committed:
Template fix

Fixed template to be empty and use .csv as output

gjusticea committed:
Internally displaced populations

Added scripts/data/metadata on internally displaced populations

gjusticea committed:
Refugees

Added script/data/metadata on refugees (forcibly displaced populations)

gjusticea committed:
Death sentences output fix

Fixed output type of death sentences list

gjusticea committed:
Started various scripts

Added starts to various scripts still pending completion

gjusticea committed:
Merge branch 'main' into add-list-scripts-greg
